rpc/types/utils.go

```diff
@@ -310,6 +283,11 @@ func AccumulativeGasUsedOfMsg(events []abci.Event, msgIndex int) (gasUsed uint64
 			continue
 		}
 
+		if len(event.Attributes) == 2 {
+			// skip the events emitted in ante handler
```

Comment:
q: how do you know these events are all from the ante handler?

Reply:
Because only the `ethereum_tx` events emitted from ante handler has exactly two attributes, the ones emitted in msg handler has more attributes.

Reply:
actually, these codes are dead now, removed.